app/controllers/clients_controller.rb

```diff
@@ -15,31 +15,23 @@ def edit; end
 
   def create
     @client = Client.new(client_params)
-
-    respond_to do |format|
-      if @client.save
+    if @client.save
+      respond_to do |format|
         format.html { redirect_to @client, notice: 'Client was successfully created' }
-        format.json { render :show, status: :created, location: @client }
-      else
-        format.html { render :new, status: :unprocessable_entity }
-        format.json { render json: @client.errors, status: :unprocessable_entity }
       end
+    else render 'new'
     end
   end
 
   def update
-    respond_to do |format|
-      if @client.update(client_params)
+    if @client.update(client_params)
+      respond_to do |format|
         format.html { redirect_to @client, notice: 'Client was successfully updated' }
```

Comment:
Remove this respond_to block from all places

Reply:
Removed